### PR TITLE
CI: deploy pipeline small fix

### PR DESCRIPTION
**Pull request by radtriste**
Many thanks for submitting your Pull Request :heart:! 

Please make sure your PR meets the following requirements:

- [ ] You have read the [contributors guide](README.md#contributing-to-kogito-images-repository)
- [ ] Pull Request title is properly formatted: `[KOGITO|RHPAM-XYZ] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a testcase that verifies it
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster
- [ ] You've added a [RELEASE_NOTES.md](RELEASE_NOTES.md) entry regarding this change

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

- <b>(Re)run Jenkins tests</b>  
  Please add comment: <b>Jenkins [test|retest] this</b>

- <b>Prod tests</b>  
  Please add comment: <b>Jenkins (re)run [prod|Prod|PROD]</b>
</details>

<details>
<summary>
How to backport a pull request to a different branch?
</summary>

In order to automatically create a **backporting pull request** please add one or more labels having the following format `backport-<branch-name>`, where `<branch-name>` is the name of the branch where the pull request must be backported to (e.g., `backport-7.67.x` to backport the original PR to the `7.67.x` branch).

> **NOTE**: **backporting** is an action aiming to move a change (usually a commit) from a branch (usually the main one) to another one, which is generally referring to a still maintained release branch. Keeping it simple: it is about to move a specific change or a set of them from one branch to another.

Once the original pull request is successfully merged, the automated action will create one backporting pull request per each label (with the previous format) that has been added.

If something goes wrong, the author will be notified and at this point a manual backporting is needed.

> **NOTE**: this automated backporting is triggered whenever a pull request on `main` branch is labeled or closed, but both conditions must be satisfied to get the new PR created.
</details>

### CODE REVIEW

File: scripts/common.py [hunk 1]
```diff
@@ -372,7 +372,7 @@ def update_maven_repo_in_behave_tests(repo_url, replace_jboss_repository):
     env_var_key = "MAVEN_REPO_URL"
     if replace_jboss_repository:
         env_var_key = "JBOSS_MAVEN_REPO_URL"
-    replacement = "| variable | value |\n      | {} | {} |\n      | MAVEN_DOWNLOAD_OUTPUT | true |".format(env_var_key,
```
Comment (radtriste) [inline scripts/common.py:375]: This is causing issue with the release when using an external repository.
This is for debugging so we don't need it normally

**Comment by kie-ci1** [comment]
**PR job** `#925` was: **ABORTED**
Possible explanation: Most probably a timeout, please review

Please look here: https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images/925/display/redirect
See console log:
<details>
<summary><b>Console Logs</b></summary>
[Pipeline] }<br/>Failed in branch kogito-jobs-service-allinone<br/>[Pipeline] }<br/>Failed in branch kogito-jobs-service-infinispan<br/>[Pipeline] }<br/>Failed in branch kogito-jobs-service-postgresql<br/>[Pipeline] }<br/>Failed in branch kogito-s2i-builder<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>Failed in branch kogito-jobs-service-ephemeral<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>Failed in branch kogito-data-index-ephemeral<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>Failed in branch kogito-jit-runner<br/>[Pipeline] // parallel<br/>[Pipeline] }<br/>[Pipeline] // script<br/>[Pipeline] }<br/>[Pipeline] // withEnv<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] stage<br/>[Pipeline] { (Declarative: Post Actions)<br/>[Pipeline] script<br/>[Pipeline] {<br/>[Pipeline] echo<br/>[INFO] Clean workspace<br/>[Pipeline] cleanWs<br/>[WS-CLEANUP] Deleting project workspace...<br/>[WS-CLEANUP] Deferred wipeout is used...<br/>[WS-CLEANUP] done<br/>[Pipeline] echo<br/>[INFO] Workspace cleaned<br/>[Pipeline] echo<br/>[INFO] Cleanup Maven artifacts<br/>[Pipeline] sh<br/>+ rm -rf /home/jenkins/.m2/repository<br/>[Pipeline] echo<br/>[INFO] .m2/repository cleaned<br/>[Pipeline] }<br/>[Pipeline] // script<br/>[Pipeline] script<br/>[Pipeline] {<br/>[Pipeline] sh<br/>+ wget --no-check-certificate -qO - https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images/925/api/json<br/>[Pipeline] readJSON<br/>[Pipeline] sh<br/>
</details>

**Comment by kie-ci1** [comment]
**PR job** `#924` was: **ABORTED**
Possible explanation: Most probably a timeout, please review

Please look here: https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images/924/display/redirect
See console log:
<details>
<summary><b>Console Logs</b></summary>
[Pipeline] // stage<br/>[Pipeline] }<br/>Failed in branch kogito-data-index-mongodb<br/>[Pipeline] }<br/>Failed in branch kogito-data-index-postgresql<br/>[Pipeline] }<br/>Failed in branch kogito-explainability<br/>Build KIE » kogito » main » pullrequest » kogito-images.build-image PR #1597 - kogito-data-index-infinispan: https://github.com/kiegroup/kogito-images/pull/1597 completed: ABORTED<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>Failed in branch kogito-data-index-infinispan<br/>Build KIE » kogito » main » pullrequest » kogito-images.build-image PR #1597 - kogito-data-index-ephemeral: https://github.com/kiegroup/kogito-images/pull/1597 completed: ABORTED<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>Failed in branch kogito-data-index-ephemeral<br/>[Pipeline] // parallel<br/>[Pipeline] }<br/>[Pipeline] // script<br/>[Pipeline] }<br/>[Pipeline] // withEnv<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] stage<br/>[Pipeline] { (Declarative: Post Actions)<br/>[Pipeline] script<br/>[Pipeline] {<br/>[Pipeline] echo<br/>[INFO] Clean workspace<br/>[Pipeline] cleanWs<br/>[WS-CLEANUP] Deleting project workspace...<br/>[WS-CLEANUP] Deferred wipeout is used...<br/>[WS-CLEANUP] done<br/>[Pipeline] echo<br/>[INFO] Workspace cleaned<br/>[Pipeline] echo<br/>[INFO] Cleanup Maven artifacts<br/>[Pipeline] sh<br/>+ rm -rf /home/jenkins/.m2/repository<br/>[Pipeline] echo<br/>[INFO] .m2/repository cleaned<br/>[Pipeline] }<br/>[Pipeline] // script<br/>[Pipeline] script<br/>[Pipeline] {<br/>[Pipeline] sh<br/>+ wget --no-check-certificate -qO - https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images/924/api/json<br/>[Pipeline] readJSON<br/>[Pipeline] sh<br/>
</details>

File: .ci/jenkins/Jenkinsfile.build-image [hunk 2]
```diff
@@ -111,13 +108,23 @@ pipeline {
...
+                }
+            }
         }
         stage('Push tags') {
             when {
                 expression { return shouldDeployImage() }
             }
             steps {
                 script {
+                    cloud.installSkopeo()
```
Comment (spolti) [inline .ci/jenkins/Jenkinsfile.build-image:118]: later, we could add skopeo in the builder.

**Review by spolti — APPROVED** [review]

File: .ci/jenkins/Jenkinsfile.build-image (hunk 2 above)
Comment (radtriste) [reply .ci/jenkins/Jenkinsfile.build-image:118]: what builder do you mean ?

Comment (spolti) [reply .ci/jenkins/Jenkinsfile.build-image:118]: jenkins builder node.

Comment (radtriste) [reply .ci/jenkins/Jenkinsfile.build-image:118]: we could indeed.

**Comment by kie-ci1** [comment]
**PR job** `#927` was: **UNSTABLE**
Possible explanation: This should be test failures

Please look here: https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images/927/display/redirect
See console log:
<details>
<summary><b>Console Logs</b></summary>
[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>Build KIE » kogito » main » pullrequest » kogito-images.build-image PR #1597 - kogito-task-console: https://github.com/kiegroup/kogito-images/pull/1597 completed: SUCCESS<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>Build KIE » kogito » main » pullrequest » kogito-images.build-image PR #1597 - kogito-swf-devmode: https://github.com/kiegroup/kogito-images/pull/1597 completed: UNSTABLE<br/>[Pipeline] unstable<br/>WARNING: Tests on kogito-swf-devmode seems to have failed<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>Build KIE » kogito » main » pullrequest » kogito-images.build-image PR #1597 - kogito-s2i-builder: https://github.com/kiegroup/kogito-images/pull/1597 completed: SUCCESS<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] // parallel<br/>[Pipeline] }<br/>[Pipeline] // script<br/>[Pipeline] }<br/>[Pipeline] // withEnv<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] stage<br/>[Pipeline] { (Declarative: Post Actions)<br/>[Pipeline] script<br/>[Pipeline] {<br/>[Pipeline] echo<br/>[INFO] Clean workspace<br/>[Pipeline] cleanWs<br/>[WS-CLEANUP] Deleting project workspace...<br/>[WS-CLEANUP] Deferred wipeout is used...<br/>[WS-CLEANUP] done<br/>[Pipeline] echo<br/>[INFO] Workspace cleaned<br/>[Pipeline] echo<br/>[INFO] Cleanup Maven artifacts<br/>[Pipeline] sh<br/>+ rm -rf /home/jenkins/.m2/repository<br/>[Pipeline] echo<br/>[INFO] .m2/repository cleaned<br/>[Pipeline] }<br/>[Pipeline] // script<br/>[Pipeline] script<br/>[Pipeline] {<br/>[Pipeline] sh<br/>+ wget --no-check-certificate -qO - https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images/927/api/json<br/>[Pipeline] readJSON<br/>[Pipeline] sh<br/>
</details>

**Comment by kie-ci1** [comment]
**PR job** `#928` was: **UNSTABLE**
Possible explanation: This should be test failures

Please look here: https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images/928/display/redirect
See console log:
<details>
<summary><b>Console Logs</b></summary>
[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>Build KIE » kogito » main » pullrequest » kogito-images.build-image PR #1597 - kogito-management-console: https://github.com/kiegroup/kogito-images/pull/1597 completed: SUCCESS<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>Build KIE » kogito » main » pullrequest » kogito-images.build-image PR #1597 - kogito-swf-devmode: https://github.com/kiegroup/kogito-images/pull/1597 completed: UNSTABLE<br/>[Pipeline] unstable<br/>WARNING: Tests on kogito-swf-devmode seems to have failed<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>Build KIE » kogito » main » pullrequest » kogito-images.build-image PR #1597 - kogito-s2i-builder: https://github.com/kiegroup/kogito-images/pull/1597 completed: SUCCESS<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] // parallel<br/>[Pipeline] }<br/>[Pipeline] // script<br/>[Pipeline] }<br/>[Pipeline] // withEnv<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] stage<br/>[Pipeline] { (Declarative: Post Actions)<br/>[Pipeline] script<br/>[Pipeline] {<br/>[Pipeline] echo<br/>[INFO] Clean workspace<br/>[Pipeline] cleanWs<br/>[WS-CLEANUP] Deleting project workspace...<br/>[WS-CLEANUP] Deferred wipeout is used...<br/>[WS-CLEANUP] done<br/>[Pipeline] echo<br/>[INFO] Workspace cleaned<br/>[Pipeline] echo<br/>[INFO] Cleanup Maven artifacts<br/>[Pipeline] sh<br/>+ rm -rf /home/jenkins/.m2/repository<br/>[Pipeline] echo<br/>[INFO] .m2/repository cleaned<br/>[Pipeline] }<br/>[Pipeline] // script<br/>[Pipeline] script<br/>[Pipeline] {<br/>[Pipeline] sh<br/>+ wget --no-check-certificate -qO - https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images/928/api/json<br/>[Pipeline] readJSON<br/>[Pipeline] sh<br/>
</details>

**Comment by kie-ci1** [comment]
**PR job** `#929` was: **UNSTABLE**
Possible explanation: This should be test failures

Please look here: https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images/929/display/redirect
See console log:
<details>
<summary><b>Console Logs</b></summary>
[Pipeline] }<br/>Build KIE » kogito » main » pullrequest » kogito-images.build-image PR #1597 - kogito-task-console: https://github.com/kiegroup/kogito-images/pull/1597 completed: SUCCESS<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>Build KIE » kogito » main » pullrequest » kogito-images.build-image PR #1597 - kogito-swf-devmode: https://github.com/kiegroup/kogito-images/pull/1597 completed: UNSTABLE<br/>[Pipeline] unstable<br/>WARNING: Tests on kogito-swf-devmode seems to have failed<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>Build KIE » kogito » main » pullrequest » kogito-images.build-image PR #1597 - kogito-s2i-builder: https://github.com/kiegroup/kogito-images/pull/1597 completed: UNSTABLE<br/>[Pipeline] unstable<br/>WARNING: Tests on kogito-s2i-builder seems to have failed<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] // parallel<br/>[Pipeline] }<br/>[Pipeline] // script<br/>[Pipeline] }<br/>[Pipeline] // withEnv<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] stage<br/>[Pipeline] { (Declarative: Post Actions)<br/>[Pipeline] script<br/>[Pipeline] {<br/>[Pipeline] echo<br/>[INFO] Clean workspace<br/>[Pipeline] cleanWs<br/>[WS-CLEANUP] Deleting project workspace...<br/>[WS-CLEANUP] Deferred wipeout is used...<br/>[WS-CLEANUP] done<br/>[Pipeline] echo<br/>[INFO] Workspace cleaned<br/>[Pipeline] echo<br/>[INFO] Cleanup Maven artifacts<br/>[Pipeline] sh<br/>+ rm -rf /home/jenkins/.m2/repository<br/>[Pipeline] echo<br/>[INFO] .m2/repository cleaned<br/>[Pipeline] }<br/>[Pipeline] // script<br/>[Pipeline] script<br/>[Pipeline] {<br/>[Pipeline] sh<br/>+ wget --no-check-certificate -qO - https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images/929/api/json<br/>[Pipeline] readJSON<br/>[Pipeline] sh<br/>
</details>

**Comment by radtriste** [comment]
Instability of tests should be solved by this PR for the downstream build-image job

**Pull request merged**